### DISCUSSION TITLE
Truffle init

### DISCUSSION THREAD

**Discussion by zaddgit** [ConsenSys-archive/truffle#6047] [Truffle Boxes]
Error: Truffle Box at URL https://github.com/truffle-box/bare-box.git doesn't exist. If you believe this is an error, please contact Truffle support.
and normally it need to creat different folders, please if someone can help with that,thanks

**Comment by FlaviuWeb3** [comment]
Hi @zaddgi please run the following command: truffle init bare-box 

If that doesn't work please let me know.

**Comment by cds-amal** [comment]
Oh noes, please use `truffle unbox bare`, the `-box` suffix is an error. 

```console

╭─me@here  ~/scratch/2023/05/16/bare-it 
╰─$ truffle unbox bare

Starting unbox...
=================

✓ Preparing to download box
✓ Downloading
✓ Cleaning up temporary files
✓ Setting up box

Unbox successful, sweet!

Commands:

  Compile:        truffle compile
  Migrate:        truffle migrate
  Test contracts: truffle test
```

**Comment by FlaviuWeb3** [comment]
I was able to run truffle unbox bare-box without any issues, is this the case for you as well?

<img width="587" alt="Screenshot 2023-05-17 at 10 58 52" src="https://github.com/trufflesuite/truffle/assets/97533870/cd88db1d-38ff-4867-8eca-9918b08a4e36">

**Comment by zaddgit** [comment]
Hi ,
Thanks for you reply, i tried it but doesn’t work ***@***.***

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

**Comment by zaddgit** [comment]
Even with the commande truffle unbox bare,

***@***.***

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows
